src-opam/opam.ml

```diff
@@ -237,12 +236,23 @@ let create_opam_branches opam_hashes =
         public_name = "opam-2.1";
         aliases = [];
       };
+      {
+        branch = "2.2";
+        hash = opam_2_2_hash;
+        enable_0install_solver = true;
+        with_vendored_deps = true;
+        public_name = "opam-2.2";
+        aliases = [];
+      };
       opam_master_branch opam_master_hash;
     ] )
 
 let create_opam_branches_windows opam_hashes =
-  let { opam_master_hash; _ } = opam_hashes in
-  (opam_master_hash, [ opam_master_branch opam_master_hash ])
+  let opam_master_hash, branches = create_opam_branches opam_hashes in
+  ( opam_master_hash,
+    List.filter
+      (fun { branch; _ } -> branch = "master" || branch = "2.2")
```

Comment:
```suggestion
      (fun { branch; _ } -> branch >= "2.2")
```

Reply:
more future proof

Reply:
ok, I'll add a comment in the code as well